docs/production-deployment/cloud/metrics/reference.mdx

```diff
@@ -26,6 +26,15 @@ keywords:
   - gRPC error count
 ---
 
+This documentation provides a guide to the metrics generated within Temporal Cloud environments.
+It enumerates the metrics, labels, and operations that enable you to analyze Workflow latencies, state transitions, gRPC errors, and more.
+
+* [Metrics labels](#metrics-labels)
+* [Available metrics](#available-metrics)
+* [Metrics groups](#metrics-groups)
+* [Metrics operations](#metrics-operations)
+* [Operations on metrics groups](#operation-ready-metrics)
+
 Most Temporal Cloud metrics are suffixed with `_count`.
```

Comment:
We kind of jump into this section a little bit.

Reply:
Agreed. Section moved down, given an H2, and intro inserted here.